### PR TITLE
Added user afh and class functionality

### PR DESCRIPTION
**Pull request by tonywu315**

### CODE REVIEW

File: constellations/gemini-admin/src/user/userAFH.js [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+
+    fetchAFHs = (afhIds) => {
+        let searchArray = new Set(afhIds);
+        API.get("api/askforhelp/all")
+            .then((res) => {
+                var userAFHs = [];
+                var afhs = [];
+                res.data.forEach((afh) => {
+                    if (searchArray.has(afh.id)) {
+                        userAFHs.push(afh);
```
Comment (mythicat554) [inline constellations/gemini-admin/src/user/userAFH.js:62]: Is afh here an afh object? Just wondering how that gets appended to an array that's for userAFHs.

Comment (tonywu315) [reply constellations/gemini-admin/src/user/userAFH.js:62]: Yeah its the afh object that contains the userId and the afhId

Comment (tonywu315) [reply constellations/gemini-admin/src/user/userAFH.js:62]: `userAFHs` is the afhs that the user is signed up for and `afhs` is all the others

**Comment by ahsu1230** [comment]
If this is going to merge into the other feature branch, you can do so whenever you want. I need to do a final review on the feature branch anyway

**Pull request merged**